.benchrc.yaml

```diff
@@ -1,10 +1,11 @@
-# Mocha opts
-extension: ["ts"]
-colors: true
-node-option:
-  - "loader=ts-node/register"
-
 # benchmark opts
 threshold: 3
-maxMs: 60_000
+maxMs: 60000
 minRuns: 10
+# Default is set to 0.005, which is too low considering the benchmark setup we have
+# Changing it to 0.05 which is 5/100, so 5% difference of moving average among run times
+convergeFactor: 0.075 # 7.5 / 100
```

Comment:
how did you come up with this number? we should probably test if this is still low enough to catch actual regressions

Reply:
I run through some runs of few files and saw the outliers. Then approximates the outlier and came up with reasonable percentage to filter. And it's just an approximation, if the outliers skewed further the benchmark can fail.